section1/reflection.md

```diff
@@ -2,18 +2,45 @@
 
```

Comment:
All your code in Section 1 looks spot on - I appreciate your attention to detail - spaces are used where they should be, semi-colons are at the end of each line, etc.